### DISCUSSION TITLE
Make gunplay great

### DISCUSSION THREAD

**Discussion by garrynewman** [Facepunch/sbox-hc1#93] [Ideas]
Here's what I would do to make the gunplay feel better

- Lots of screen shake
- Less aim cone
- Recoil knocks view about
- Can run with shift, which lowers gun
- When you stop running it takes half a second or so to bring gun back up
- Aim down sight right click all guns, most accuracy, reduce recoil

**Comment by ryleigh** [comment]
yeah i would enjoy aiming down sights, and moving the view around with recoil would help me understand where my bullets are going

**Comment by benobox** [comment]
100% agree on all of those.
Maybe a staged reload system as well to allow resuming a reload after a quick weapon swap.
Would also like a fire rate toggle for appropriate weapons

**Comment by Metapyziks** [comment]
So Call of Duty? 😛

**Comment by DevulTj** [comment]
First batch of changes made here - https://github.com/Facepunch/sbox-hc1/issues/97, we'll playtest it today and see if it's a good move.

**Reply by garrynewman** [reply]
Looking forward to playest!
